repl/src/main/resources/fake_shell.py

```diff
@@ -338,6 +340,29 @@ def magic_json(name):
         'application/json': value,
     }
 
+def magic_matplot(name):
+    try:
+        value = global_dict[name]
+    except KeyError:
```

Comment:
This is not your fault, but this exception handling is just copy & pasted everywhere... it would be much better to just have a single exception handler in `MagicNode`. You don't need to worry about that though.

But in your case, you can have a single `try..except` for the whole method, instead of the current two.